src/constants.js

```diff
@@ -1,3 +1,7 @@
+/*
+    Use constants when .env file not specified
```

Comment:
👍 for the comment. But we prefer the following comment style in our codebase:
```js
// NOTE: line 1
// line 2
...
// line n
```
or
```js
// HACK:
// ...
```
or
```js
// TODO:
// ...
```